tests/data/codelist/tier_attribute/valid/tag_fuel.yaml

```
@@ -0,0 +1,4 @@
+- Fuel:
+  - Coal:
+      description: coal
+      tier: ^1
```

Suggest to test both cases here: no change and increment by 1
```suggestion
  - Coal|Lignite:
      tier: ^1

```